scripts/test_android_unit.sh

```diff
@@ -2,5 +2,8 @@
 
 ANDROID_PLUGIN_ROOT="$PROJECT_ROOT/unity_buy_android"
 
+# Run checkstyle before running tests
+"$ANDROID_PLUGIN_ROOT"/gradlew unity_buy_plugin:checkstyle -p "$ANDROID_PLUGIN_ROOT" || die "Failed to run checkstyle"
+
```

Comment:
You can do it inside of gradle, so that you don't need this line.

```
afterEvaluate {
      if (project.tasks.findByName('check')) {
        check.dependsOn('checkstyle')
      }
    }
```